src/psyclone/tests/psyir/symbols/symboltable_test.py

```diff
@@ -1400,7 +1401,7 @@ def test_symbol_from_tag():
     with pytest.raises(SymbolError) as err:
         symtab.symbol_from_tag("tag3", symbol_type=RoutineSymbol)
     assert ("Expected symbol with tag 'tag3' to be of type 'RoutineSymbol' "
-            "but found type 'DataSymbol'." in str(err))
+            "but found type 'DataSymbol'." in str(err.value))
```

Comment:
Thanks for fixing. Is there some way we can reproduce the failure in the test suite on GitHub Actions so that we don't keep making this mistake?